tools/bridge/tests/test_config.py

```diff
@@ -3,18 +3,64 @@
 from bridge.config import validate_config
 
 
-def test_validate_missing_keys():
+@pytest.fixture(scope="session")
+def valid_config():
+    return {
+        "home_rpc_url": "url",
+        "foreign_rpc_url": "url",
+        "token_contract_address": "0x4B0b6E093a330c00fE614B804Ad59e9b0A4FE8A9",
+        "foreign_bridge_contract_address": "0xbb1046b0fe450aa48deaff6fa474adbf972840dd",
+    }
+
+
+def test_validate_missing_keys(valid_config):
+    invalid_config = {**valid_config}
+    del invalid_config["home_rpc_url"]
+
     with pytest.raises(ValueError):
-        validate_config(
-            {
-                "home-rpc-url": "url",
-                # "foreign-rpc-url": missing,
-            }
-        )
+        validate_config(invalid_config)
+
+
+def test_validate_unknown_keys(valid_config):
+    with pytest.raises(ValueError):
+        validate_config({**valid_config, "unknown_key": 1})
+
+
+def test_validate_invalid_rpc_url(valid_config):
+    with pytest.raises(ValueError):
+        validate_config({**valid_config, "home_rpc_url": 1})
```

Comment:
This should be a test for `validate_rpc_url`, not for `validate_config`.

Reply:
Same for tests below.

Reply:
:heavy_check_mark:

Reply:
Also added positive tests for them.